api/src/test/java/org/openmrs/module/dhisconnector/api/DHISConnectorServiceTest.java

```diff
@@ -186,4 +186,36 @@ public void testDHISMappingExport() throws IOException {
 				dhisConnectorService.exportMappings(new String[]{"mapping-test-unit." + mapping.getCreated()});
 		Assert.assertEquals("Successfully bundled the mapping with the metadata", pathToBundle[0]);
 	}
+
+	@Test
+	public void testDHISMappingImport() throws IOException {
+		DHISConnectorService dhisConnectorService = Context.getService(DHISConnectorService.class);
+		ReportDefinitionService reportDefinitionService = Context.getService(ReportDefinitionService.class);
+
+		DHISMapping mapping = new DHISMapping();
+		mapping.setName("mapping-test-unit");
+		mapping.setCreated(new Date().getTime());
+		PeriodIndicatorReportDefinition pir = new PeriodIndicatorReportDefinition();
+		final UUID pirUuid = UUID.randomUUID();
+		final String uuidString = String.valueOf(pirUuid);
```

Comment:
Did you also take a look at our naming conventions? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
Sure, I'll make them Uppercase, thank you

Reply:
Done